styleguide/source/assets/js/webforms.js

```diff
@@ -64,6 +64,14 @@
 
   Drupal.behaviors.webForm = {
     attach: function (context, settings) {
+
+      $(".ama__sales-landing-page__form form").validate({
+        success: function(element) {
+          $('.ama__sales-landing-page__form__heading').hide();
```

Comment:
This seems to be firing too often, I'm not sure we can rely on validate. I was messing around with using the `detach` behavior in conjunction with `attach`. See if it works alright for you https://gist.github.com/tcfunk/9a10f63a0785281b9ac57393d38c1a4d